src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -43,72 +38,31 @@ class ScalaSettings extends Settings.SettingGroup {
   val nobootcp = BooleanSetting("-nobootcp", "Do not use the boot classpath for the scala jars.")
   val strict = BooleanSetting("-strict", "Use strict type rules, which means some formerly legal code does not typecheck anymore.")
 
-  val argfiles = BooleanSetting("@<file>", "A text file containing compiler arguments (options and source files)")
```

Comment:
This is important for Windows, which has a limit on the command-line length. Without this option it is impossible to compile the Scala standard library on Windows, for example, because there are too many files to list on the command line.